### PR TITLE
+doc more details on akka.actor.Status support in ask

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/actors.md [hunk 1]
```diff
@@ -745,8 +745,12 @@ is available in the `akka.pattern.PatternsCS` object.
 
 @@@ warning
 
-To complete the future with an exception you need send a Failure message to the sender.
-This is *not done automatically* when an actor throws an exception while processing a message.
+To complete the future with an exception you need send an `akka.actor.Status.Failure` message to the sender.
```
Comment (sebastianharko) [inline akka-docs/src/main/paradox/scala/actors.md]: small typo "you need to send" instead of "need send"

Comment (ktoso) [inline akka-docs/src/main/paradox/scala/actors.md]: Thanks!

File: akka-docs/src/main/paradox/scala/actors.md [hunk 2]
```diff
@@ -745,8 +745,12 @@ is available in the `akka.pattern.PatternsCS` object.
 
 @@@ warning
 
-To complete the future with an exception you need send a Failure message to the sender.
-This is *not done automatically* when an actor throws an exception while processing a message.
+To complete the future with an exception you need to send an `akka.actor.Status.Failure` message to the sender.
+This is *not done automatically* when an actor throws an exception while processing a message. 
+
+Please note that Scala's `Try` sub types `scala.util.Failure` and `scala.util.Success` are not treated 
+specially, and would complete the ask Future with given value - only the `akka.actor.Status` messages
```
Comment (sebastianharko) [inline akka-docs/src/main/paradox/scala/actors.md]: small typo: "with *the* given value" instead of "with given value"

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Thanks for pushing it over the finish line Johan 👍

**Pull request merged**